### PR TITLE
support squeeze

### PR DESCRIPTION
**Pull request by eaidova**

### CODE REVIEW

File: src/frontends/pytorch/src/utils.hpp [hunk 1]
```diff
@@ -84,6 +84,21 @@ OutputVector translate_1to1_match_2_inputs(NodeContext& context) {
...
+template <typename T>
+OutputVector translate_1to1_match_1_or_2_inputs(NodeContext& context) {
+    auto inputs = context.inputs();
+    FRONT_END_OP_CONVERSION_CHECK(inputs.size() >= 1, "Operation has no inputs.");
+    for (int i = 2; i < inputs.size(); i++) {
+        FRONT_END_OP_CONVERSION_CHECK(context.input_is_none(i), "Got more inputs than expected.");
+    }
+    FRONT_END_OP_CONVERSION_CHECK(!context.input_is_none(0), "Input 1 should not be None.");
+    if (context.input_is_none(1)){
+    return {context.mark_node(std::make_shared<T>(inputs[0]))};
```
Comment (mvafin) [inline src/frontends/pytorch/src/utils.hpp]: Code formatting is weird here...

Comment (eaidova) [reply src/frontends/pytorch/src/utils.hpp]: refactored

File: src/frontends/pytorch/src/op/squeeze.cpp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset8.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_squeeze(NodeContext& context) {
+    auto inputs = context.inputs();
```
Comment (mvafin) [inline src/frontends/pytorch/src/op/squeeze.cpp:14]: Please validate that there is at least 1 input in `inputs`

Comment (eaidova) [reply src/frontends/pytorch/src/op/squeeze.cpp:14]: added checks

**Review by mvafin — APPROVED** [review]

**Pull request merged**